.github/workflows/ghstack_land.yaml

```diff
@@ -0,0 +1,48 @@
+name: Land ghstack PRs
+on:
+  repository_dispatch:
+    types: [land-command]
+
+jobs:
+  ghstack_land:
+    name: ghstack Land
+    runs-on: ubuntu-latest
+    steps:
+    - name: Show Environment Variables
+      run: env
+    - name: Show Github Object
+      run: |
+        cat <<'EOF'
+        ${{ toJson(github) }}
+        EOF
+    - name: Show Github Event Path Json
+      run: 'cat $GITHUB_EVENT_PATH || true'
+    - uses: actions/checkout@v3
+      with:
+        token: ${{ secrets.LAND_TOKEN }}
+        fetch-depth: '0'
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+    - name: Check Current CI Status
+      run: |
+        pip install requests ghstack
+        .github/workflows/scripts/ghstack-perm-check.py <<'EOF'
+        ${{ toJson(github) }}
+        EOF
+      env:
+        GITHUB_TOKEN: ${{ secrets.LAND_TOKEN }}
+    - name: Land It!
+      run: |
+        git config --global user.email "andrea.frittoli@gmail.com"
+        git config --global user.name "Andrea Frittoli"
```

Comment:
shouldn't the user data be `fms-cibot` etc?

Reply:
Yes, of course, sorry, copy paste issue, should be fixed now